lang/python/python/Makefile

```diff
@@ -165,14 +165,12 @@ define Build/Prepare
 	$(CP) ./files/config.site $(PKG_BUILD_DIR)/config.site
 endef
 
-ifdef CONFIG_PACKAGE_python-setuptools
```

Comment:
i would leave this out; it would save some build time ; 
this is setuptools on the target-side, which may be non-needed 

setuptools & pip on the host side are always installed [via std Python build, since --ensure-pip is set to install]
Python packages require setuptools on the host [during build], not on the target